### ISSUE TITLE
question for parameter setting 

### ISSUE THREAD

**Issue by lai-agent-t** [ymcui/Chinese-BERT-wwm#127]
What's your weight decay, max grad norm, adam epsilon and gradient accumulation steps for your first training roberta bert?

**Comment by ymcui** [comment]
1. We do not use gradient accumulation
2. For other params, we follow the default settings.  WD=0.01, norm=1.0, eps=1e-6
see: https://github.com/google-research/bert/blob/master/optimization.py#L61

**Comment by lai-agent-t** [comment]
Really Thanks!
